src/duplicacy_b2client.go

```diff
@@ -299,6 +323,18 @@ func (client *B2Client) ListFileNames(startFileName string, singleFile bool, inc
 
 		output := B2ListFileNamesOutput{}
 
+		if !inputMapBool && singleFile && !includeVersions {
```

Comment:
Do we need to test !inputMapBool here? Is there a situation where we'd get here and singleFile would be true, includeVersions would be false, and inputMapBool would be true? Seems like unnecessary complexity to the if statement.

Reply:
You're right, it's there as a sanity check, but it is redundant. That bool is going away regardless. I'll see if it makes sense to have anything similar after the other changes.